.storybook/routes.js

```diff
@@ -37,13 +38,16 @@ const routes = {
   '/components/navigation/breadcrumbs/': 'components-navigation-breadcrumbs-react--basic',
   '/components/navigation/pagination/': 'components-navigation-pagination-react--basic',
   '/components/popups/toast/': 'components-popups-toast-react--with-popper',
-  '/components/popups/menu/': 'preview-menu-react--basic',
+  '/components/popups/menu/': 'components-popups-menu-react--basic',
```

Comment:
We should consider updating the Menu page on the Canvas Site to display docs for the Menu in Main. [Currently](https://canvas.workday.com/components/popups/menu), it displays docs for the Menu in Preview.

Reply:
We've decided to update the Menu page on the Canvas Site to display docs for the Menu in Main.